test/behaviors/FifoCrcMarket.js

```diff
@@ -84,10 +84,11 @@ const shouldBehaveLikeFifoCrcMarketV0 = admin => {
           for (let i = 0; i < crcToMint; i++) {
             crc.mint(supplier, '', token(crcValue), '');
           }
+
           suppliersCrcBal += crcToMint;
           assert.equal(
             await crc.balanceOf(supplier),
-            suppliersCrcBal,
+            token(suppliersCrcBal),
```

Comment:
where is `token()` defined?

Reply:
one of the helper files. It basically just is a shortcut for web3.toWei